xml/System.Data.Spatial/DbGeometry.xml

```diff
@@ -860,8 +860,8 @@
         <ReturnType>System.Nullable&lt;System.Int32&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the number of interior rings in this DbGeometry value, if it represents a polygon. &lt;returns&gt;The number of elements in this geometry value, if it represents a polygon; otherwise <see langword="null" />.&lt;/returns&gt;</summary>
-        <value>The number of interior rings in this DbGeometry value.</value>
+        <summary>Gets the number of interior rings in this DbGeometry value, if it represents a polygon.</summary>
+        <value>The number of elements in this geometry value, if it represents a polygon; otherwise <see langword="null" />.</value>
```

Comment:
```suggestion
        <value>The number of elements in this geometry value, if it represents a polygon; otherwise, <see langword="null" />.</value>
```